generators/craft3/index.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+var Generator = require('yeoman-generator');
+var chalk = require('chalk');
+var yosay = require('yosay');
+var del = require('del');
+var fs = require('fs');
+var child_process = require('child_process');
+var prompts = require('./modules/prompts');
+var plugins = require('./modules/craft_plugins');
+
+module.exports = Generator.extend({
+  initializing: function() {
+    this.closingStatements = [];
+  },
+
+  prompting: function() {
+    return this.prompt(prompts).then(function(options) {
+      Object.assign(this.options, options);
+    }.bind(this));
+  },
+
+  writing: {
+    setRoot: function() {
+      this.destinationRoot('./');
+    },
+
+    downloadCraft: function() {
+      child_process.execSync(`composer create-project -s RC craftcms/craft ${this.options.projectName}-craft`);
```

Comment:
I'm guessing that RC will go away relatively shortly. Worth adding a flag for it now so we don't have to change this later?

Reply:
Good point. But we won't have to deal with this for 4 months anyway, and we'll probably want to manually bump the `RCx` version in `composer.json` every once in awhile too, so I'm not too worried about it. It only affects the generator itself, not projects generated by it.